src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,126 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import { FC } from 'react';
+import { Person } from '../types';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { PersonLink } from './PersonLink';
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const selected = people.find(person => person.slug === slug);
+
+  const currentSort = searchParams.get('sort');
+  const currentOrder = searchParams.get('order');
+
+  const getNewSortParams = (field: string) => {
+    if (currentSort === field && currentOrder === 'asc') {
+      return { sort: field, order: 'desc' };
+    } else if (currentSort === field && currentOrder === 'desc') {
+      return { sort: null, order: null };
```

Comment:
Returning `null` for `sort` and `order` in `getNewSortParams` might not remove these parameters from the URL. Consider returning an empty object or explicitly removing these parameters to ensure they are reset properly.